crates/alpen-ee/common/src/traits/prover.rs

```diff
@@ -15,8 +15,6 @@ pub enum ProofGenerationStatus {
 }
 
 /// Interface between Prover and Batch assembly
-/// Note: In case of reorgs, `EeUpdateId` is unique enough to not conflict between forks.
```

Comment:
This docstring (the "Note:") was valuable.
The one who as not was the "Note2: " 😂